src/java.net.http/share/classes/jdk/internal/net/http/ResponseBodyHandlers.java

```diff
@@ -56,7 +52,7 @@ public final class ResponseBodyHandlers {
 
     private ResponseBodyHandlers() { }
 
-    private static final String pathForSecurityCheck(Path path) {
+    private static final String pathForDefaultFSCheck(Path path) {
```

Comment:
maybe this method and the equivalent in the previous file could be renamed `checkPathForDefaultFS` which would suggest an exception being thrown in error cases.

Reply:
Good idea for the renaming. I will change that - and above too.